docs/index.md

```diff
@@ -1,40 +1,68 @@
+
+NeIC Sensitive Data Archive
+===========================
+
+The NeIC Sensitive Data Archive (SDA) software is a modular system of microservices that in alternative configurations can be setup to host sensitive data in an archival service, securely stored encrypted at rest.
```

Comment:
To me, this sentence is a bit complex for a first introduction to the project. Maybe something like:
```suggestion
The NeIC Sensitive Data Archive (SDA) is an encrypted data archive, originally implemented for storage of sensitive biological data. It is implemented as a modular microservice system that can be deployed in different configurations depending on the service needs.
```
would be easier to read for new users?